pl_examples/pytorch_ecosystem/pytorch_geometric/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
```

Comment:
Is this file needed for the example? The less files the better imo

Reply:
Install with Pytorch Geometric can be pretty complicated. `poetry install` makes it simpler.